test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,31 @@
+require "test_helper"
+
+describe RecipesController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
+  describe "homepage" do
+    it "can get search page" do
+      get root_path
+       must_respond_with :success
+    end
+  end
+
+  describe "index" do
+    it "can get a list of recipes" do
```

Comment:
You should also have a test a search result with nonsense like `^&*` and "".  In other words include negative cases.